ios/FluentUI/Button/Button.swift

```diff
@@ -106,7 +114,7 @@ open class Button: UIButton, TokenizedControlInternal {
             configuration.contentInsets = edgeInsets
             let titleTransformer = UIConfigurationTextAttributesTransformer { incoming in
                 var outgoing = incoming
-                outgoing.font = UIFont.fluent(self.tokenSet[.titleFont].fontInfo)
+                outgoing.font = UIFont.fluent(self.tokenSet[.titleFont].fontInfo, shouldScale: false)
```

Comment:
Why do we need to set the shouldScale here?

Reply:
We don't, I accidentally left that in from when I was trying to figure something else out. Thanks!